### PR TITLE
Press

### PR DESCRIPTION
**Pull request by mcmikashi**
I create a new app press and created 3 views :

1. ListView
2. CreateView
3. View

And I add listview and createview on the dashboard

### CODE REVIEW

**Comment by JeffAbrahamson** [comment]
Very well done, thanks!

Suggestions:
* Maybe be more positive on the rights protection, like "Toujours respecter (respectons ?) les droits d'auteur" or something like that?
* The submit button might be "Sauvegarder", since we're not really sending anything.
* In the /new form, the names are in English but the users will be French.
* Suggestion for display: Publication, URL, Titre, Resume
* Dates are formatting in English for me, but perhaps that's just my browser being too ready to consider English
* The `/view` page is gonna need work, but don't sweat that now.
* In another commit, maybe we could add tests?  ;-)

@Shriukan33 Any other comments?

**Review by Shriukan33 — CHANGES_REQUESTED** [review]
The responsive needs some tweaking here : 
image

Overall it's a good job, and I agree with @JeffAbrahamson  comments, especially the possibility to edit the record would be nice, also delete an existing record would be nice. 

Last, the buttons could use the .donationtion-button / .navigation-button classes

File: transport_nantes/press/models.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    class Meta:
+        permissions = (
+            # Allow the user to see list view and create article
+            ("press-editor", "May create and see list view Article"),
+        )
+    name = models.CharField(max_length=200)
+    link = models.URLField(max_length=255)
+    title = models.CharField(max_length=200)
+    summary = models.TextField()
+    publication_date = models.DateField(auto_now_add=True)
```
[the remark this answers is not in the document]
Comment (JeffAbrahamson) [reply transport_nantes/press/models.py:14]: Journal publication date.

File: transport_nantes/press/models.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+
+class Article(models.Model):
+    class Meta:
+        permissions = (
+            # Allow the user to see list view and create article
+            ("press-editor", "May create and see list view Article"),
+        )
+    name = models.CharField(max_length=200)
```
[the remark this answers is not in the document]
Comment (JeffAbrahamson) [reply transport_nantes/press/models.py:10]: I think `name` is the journal name: Le Monde, Le Figaro, Ouest France, etc.

This is why it's good to assign more descriptive variable names than simple common nouns and verbs, especially if the variable will be used across more than 3-4 contiguous lines of code.

**Comment by JeffAbrahamson** [comment]
@mcmikashi I think you should stabilise the commit you've got, get it merged, then improve it.  So that means things like names of stuff, url paths, etc.

But buttons and editing and responsiveness I think you can do in a second commit.

File: transport_nantes/press/models.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Shriukan33) [reply transport_nantes/press/models.py:14]: Then this field wont do, it only adds the current datetime.now(), as the record is created.

File: transport_nantes/press/models.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Shriukan33) [reply transport_nantes/press/models.py:10]: Ok, then the admin panel display shouldn't only display the journal's name, as it's not descriptive enough to find what you want.

File: transport_nantes/press/templates/press/table_list.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            <tr>
+                <td>{{ article.name }}</td>
+                <td>{{ article.title }}</td>
+                <td>{{ article.summary }}</td>
+                <td>{{ article.publication_date }}</td>
+                <td><a class="btn btn-outline-info" href="{{article.link }}" role="button" target="_blank">Lien</a></td>
+            </tr>
+        {% endfor %}
+    </tbody>
+</table>
```
Comment (Shriukan33) [inline transport_nantes/press/templates/press/table_list.html:22]: Missing EOL :)

**Comment by JeffAbrahamson** [comment]
In the table list, you say
```
+    <tr>
+      <th scope="col">Nom</th>
+      <th scope="col">Lien</th>
+      <th scope="col">Titre</th>
+      <th scope="col">Description</th>
+      <th scope="col">Date</th>
+    </tr>
```

Perhaps nom -> journal, lien -> URL ?

**Comment by JeffAbrahamson** [comment]
Otherwise, LGTM.

**Pull request merged**